frame/transaction-payment/src/lib.rs

```diff
@@ -233,6 +247,9 @@ impl<M: Get<Multiplier>> MultiplierUpdate for ConstFeeMultiplier<M> {
 	fn min() -> Multiplier {
 		M::get()
 	}
+	fn max() -> Multiplier {
+		<Multiplier as sp_runtime::traits::Bounded>::max_value()
```

Comment:
```suggestion
		M::get()
```

This struct should implement `MultiplierUpdate` such that it always returns a constant value. In this case, I think you just want to return `M::get()` like `min()` does.

Reply:
Shouldn't this return the maximum value?